test/unit/test-kernel.cpp

```diff
@@ -918,7 +918,129 @@ TEST(Kernel, CollapseSeq)
   delete[] x;
 }
 
+//Sequential region
+TEST(Kernel, RegionSeq)
+{
+
+  const int N = 300;
+  int * Arr_a = new int[N];
+  int * Arr_b = new int[N];
+  int * Arr_c = new int[N];
+
+  for(int i=0; i<N; ++i) {
+    Arr_a[i] = 0;
+    Arr_b[i] = 0;
+    Arr_c[i] = 0;
+  }
+
+  using Pol =
+    RAJA::KernelPolicy<
+      RAJA::statement::Region<RAJA::seq_region,
+
+        RAJA::statement::For<0, RAJA::loop_exec,
+                           RAJA::statement::Lambda<0>
+        >
+
+       ,RAJA::statement::For<0, RAJA::loop_exec,
```

Comment:
Please format this policy so it's easier to read; e.g., indentation, vertical alignment, and move commas to end of preceding line.